.idea/misc.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should ignore `.idea` folder by using `.gitignore`. The folder auto generated by PHPstorm